src/app/components/core/developer/packages/add-python-package/add-python-package.component.html

```diff
@@ -0,0 +1,80 @@
+<div class="container is-fluid">
+  <div class="navbar-item">
+    <nav class="breadcrumb" aria-label="breadcrumbs">
+      <ul>
+        <li><a [routerLink]="['/developer']">Developer</a></li>
+        <li><a [routerLink]="['/developer/python/package/list']">Python Packages</a></li>
+        <li class="is-active">
+          Add
+        </li>
+      </ul>
+    </nav>
+  </div>
+  <div class="card">
+    <div class="card-content step-content">
+      <form>
+        <div class="column is-four-fifths">
+          <div class="field is-horizontal">
+            <div class="field-label">
+              <label class="label">Package Name</label>
+            </div>
+            <div class="field-body">
+              <div class="field">
+                <div class="control">
+                  <input name="name" class="input is-small" type="text" placeholder="Package" required>
+                </div>
+
+                <!-- <div *ngIf="name.invalid && (name.dirty || name.touched)" class="help is-danger">
+                  <div *ngIf="name.errors?.['required']">
+                    Name is required.
+                  </div>
+                </div> -->
+              </div>
+
+            </div>
+          </div>
+          <div class="field is-horizontal">
+            <div class="field-label">
+              <label class="label">Version</label>
+            </div>
+            <div class="field-body">
+              <div class="field">
+                <div class="control">
+                  <input name="name" class="input is-small" type="text" placeholder="Version">
```

Comment:
placeholder="1.0.0"

Reply:
also can we make 1/2 or 2/3 of current input size? May be we can simply have a pop-up unless operation is async.